README.md

```diff
@@ -1,11 +1,15 @@
 # Carolina
 
-Carolina is a [pyDAKOTA](https://github.com/wisdem/pyDAKOTA) fork maintained by Equinor.  Its raison d'être is to have easier building of a Python [Dakota](https://dakota.sandia.gov/) wrapper, without any MPI support.
+Carolina is a [pyDAKOTA](https://github.com/wisdem/pyDAKOTA) fork maintained by Equinor.  Its raison d'être is to have easier building of a Python [Dakota](https://dakota.sandia.gov/) wrapper, without any MPI support. Carolina supports Python version 3.6, 3.7, 3.8, 3.10
```

Comment:
Please remove 3.6 and 3.7. Yes, this was supported once, but do we really guarantee that this is still supported?

Reply:
Removed